precompiled/Kick

```diff
@@ -25,11 +25,14 @@ if [ "$UNAME" == "Darwin" ]; then
 		export DYLD_INSERT_LIBRARIES="$STEAM_DYLD_INSERT_LIBRARIES"
 	fi
 
+	ln -sf mcs.bin.osx mcs
 	./kick.bin.osx $@
 else
 	if [ "$ARCH" == "x86_64" ]; then
+		ln -sf mcs.bin.x86_64 mcs
```

Comment:
While this works on Steam, this could be a problem in the long term. In particular, GOG/MojoSetup builds will NOT be a fan of this - if a game is installed in a location not normally accessible by the user (i.e. non-root) then the ln call will fail. If the game really depends on `mcs`, it's probably safer to make 32/64-bit builds separate, with mcs being the native mkbundle binary renamed, skipping the need for symlinks. In AppAdmin this would simply be 2 depots limited to Linux32 and Linux64 (with actual game content in the existing Linux multiarch depot), containing the Kick binaries exclusive to those architectures, and that same layout can be applied to MojoSetup packages without runtime symlinking.

It sounds like a PITA, but so is this whole XML incompatibility... MONOOOOOOOOO :shakefist: